src/methods/alphabetical-sort.js

```diff
@@ -34,8 +34,11 @@ const alphabeticalSort = list => {
  * @returns {array}
  *
  * @example
- * // TODO
- *
+ * // Sample array with name or title's property
+ * ecomUtils.alphabeticalSort(["product", "category"])
+ * => ["category", "product"]
```

Comment:
you should mark output with comment, as it's not a valid JS code, right?
```suggestion
 * // => ["category", "product"]
```